en/includes/references/email-templates.md

```diff
@@ -229,13 +229,20 @@ The following literals about the user are accessible for all email templates.
             <td>{{"{{ userstore-domain }}"}}</td>
             <td>Name of the user store.</td>
         <tr>
+        <tr>
+            <td>{{"{{ organization-name }}"}}</td>
+            <td>Name of the organization. Insert this placeholder where you want to display the organization's name in a human readable format.</td>
+        </tr>
         <tr>
             <td>{{"{{ tenant-domain }}"}}</td>
-            <td>Name of the organization.</td>
+            <td>Domain name specific to the organization. Utilize this placeholder within URL paths to denote the organization's unique ID.</td>
```

Comment:
tenant domain of root orgs are human readble 
tenant domain value of sub orgs are UUID
so can we define this in a different manner?

Reply:
Suggestion:

`Domain name specific to the organization. For root organizations, this is a human-readable domain name. For sub-organizations, a UUID is used to uniquely identify them. Utilize this placeholder within URL paths to denote the organization's ID.`

Reply:
Utilize this placeholder within URL paths to denote the tenant. WDYT?